docs/CONTRIBUTING.md

```diff
@@ -120,7 +120,7 @@ Before running the tests, make sure you have a working API and app deployed to t
 
 Once you have a working API and app deployed to the cloud, make sure to open the `cypress.json` in the project root, and configure all of the variables.
 
-Most of the needed values can be found in the `.env.json` files in your `sample-project/apps` folders (e.g. `sample-project/apps/admin/.env.json`).
+Most of the needed values can be found in the `.env.json` files in your `apps` folders (e.g. `apps/admin/.env.json`).
```

Comment:
This is no longer true, as we inject ENV values at runtime. So let's remove this sentence completely for now, until we create that CLI command to view project info #1143.